PF/PF-Assignment-1/Shopping.java

```diff
@@ -0,0 +1,132 @@
+package sc;
+import java.util.ArrayList;
+import java.util.List;
+public class Shopping {
+       
+    private List<Item> Cart = new ArrayList<Item>();
+    
+    ShoppingCart(){
+    	
+    }
+ 
+    public void addItem(String Pname,float Pprice,int Pquantity )
+    {
+        Item Product= new Item(Pname,Pprice,Pquantity);
+        Cart.add(Product);
```

Comment:
Like here, it looks like a class named Cart and a static method is getting called on it, instead it's a variable name